addons/html_editor/static/src/editor/core/hint_plugin.js

```diff
@@ -111,17 +120,13 @@ export class HintPlugin extends Plugin {
     }
 
     makeHint(el, text) {
-        this.shared.disableObserver();
         el.setAttribute("placeholder", text);
         el.classList.add("o-we-hint");
-        this.shared.enableObserver();
```

Comment:
Wait this is not needed anymore ?? :exploding_head: :heart_on_fire:

Reply:
I'll try as hard as I can to not re-introduce enable/disable observer/pause/stack/...

Reply:
You're a :crown: